### PR TITLE
NTP-479: Error message when you don't confirm cookie opt in

### PR DESCRIPTION
**Pull request by stephenjmcneill**
## Context

So that a user cannot save a cookies option without first selecting an option. 

## Changes proposed in this pull request

**Before:**

Error Banner before

**After:**

Error Banner after

## Link to Jira ticket

https://dfedigital.atlassian.net/browse/NTP-479

## Things to check

- [x] Title is in the format "NTP-XXX: Short description" where NTP-XXX is the Jira ticket reference
- [x] Code and tests follow the [coding standards](/docs/coding-standards.md)
- [x] `dotnet format` has been run in the repository root
- [x] Test coverage of new code is at least 80%
- [x] All UI related acceptance criteria specified in the ticket have been added to the relevant [cypress test feature file(s)](/UI/cypress/e2e/)
- [ ] Unless explicitly mentioned in the ticket, all UI work should have been tested as working on desktop and mobile resolutions with JavaScript on and off
- [ ] Database migrations can be applied to the current codebase in main without causing exceptions
- [ ] Debug logging has been added after all logic decision points
- [x] All [automated testing](/README.md#testing) including accessibility and security has been run against your code

### CODE REVIEW

File: UI/cypress/e2e/cookies.feature [hunk 1]
```diff
@@ -70,5 +70,11 @@ Scenario: Allow users to opt out once they have reached the cookies page for fir
     And a Success Banner is displayed
     And user session is not tracked
 
+Scenario: Error banner appears if no option is selected
+    Given the 'view cookies' page is displayed
+    And nothing is selected
+    When Saves Changes
+    And Saves Changes
```
Comment (iterative-it) [inline UI/cypress/e2e/cookies.feature:77]: I think this and step can be removed

File: UI/cypress/e2e/cookies.feature [hunk 2]
```diff
@@ -70,5 +70,11 @@ Scenario: Allow users to opt out once they have reached the cookies page for fir
     And a Success Banner is displayed
     And user session is not tracked
 
+Scenario: Error banner appears if no option is selected
+    Given the 'view cookies' page is displayed
```
Comment (iterative-it) [inline UI/cypress/e2e/cookies.feature:74]: There's some inconsistencies in this file in the use of single quotes. Please can you change all the unicode ‘ and ’ characters to the ASCII ' character

**Review by iterative-it — CHANGES_REQUESTED** [review]

File: UI/cypress/e2e/cookies.feature (hunk 2 above)
Comment (stephenjmcneill) [reply UI/cypress/e2e/cookies.feature:74]: Done

File: UI/cypress/e2e/cookies.feature (hunk 1 above)
Comment (stephenjmcneill) [reply UI/cypress/e2e/cookies.feature:77]: Removed extra 'Save changes' step

**Review by iterative-it — APPROVED** [review]

**Pull request merged**